src/main/java/frc/robot/RobotContainer.java

```diff
@@ -90,6 +96,7 @@ private void configureBindings() {
     driverController.x().onTrue(enterXMode);
     driverController.a().onTrue(runIntake);
     driverController.y().onTrue(deployPneumatics);
+    driverController.rightTrigger().onTrue(deployUrMom);
```

Comment:
...tbh let's not bind a button to run this command. We'll just wind up unbinding it later when we inevitably need that button for something actually useful